lib/github-queue.js

```diff
@@ -7,8 +7,24 @@ const statsd = require('./statsd')
 const getToken = require('./get-token')
 const Github = require('../lib/github')
 
-const writeQueue = new Queue(1, Infinity)
-const readQueue = new Queue(50, Infinity)
+const readQueues = {}
+const writeQueues = {}
+
+function getWriteQueue (installationId) {
+  // requests per user should be serial
+  // hence the queue concurrency of 1
+  const writeQueue = writeQueues[installationId] || new Queue(1, Infinity)
+  writeQueue[installationId] = writeQueue
```

Comment:
Shouldn’t it be `writeQueue__s__[installationId] = writeQueue`? Same below for reads. Either way, assigning ` writeQueues[installationId]` to `writeQueue` and, if it exists, straight back into `writeQueues[installationId]` seems a bit unnecessary.